#168364828  add datase configuration

ramadhanbridge opened the pull request:
#### What does this PR do?
> Add database to persist data;
#### Description of Task to be completed?
> Whith this PR, I added data base configuration and I include it, in all filles that will require to use database
#### How should this be manually tested?
.  you can clone my project
.  you can use links that I provided on readme.md
#### Any background context you want to provide?
> N/A
#### What are the relevant pivotal tracker stories?
> database setup, with story id of #168364828
#### Screenshots (if appropriate)
![Capture10](https://user-images.githubusercontent.com/38207385/64549293-b6c89380-d330-11e9-872d-cb1b25d91d53.PNG)

ramadhanbridge committed:
chore(database): add datase configuration
>
>  I added datase in this files
> I changed db files from memory to db
>
> [finishes #168364828]

Oluwasegun-AA commented on server/models/db.js:
```diff
@@ -0,0 +1,35 @@
+
+     CREATE TABLE IF NOT EXISTS sessions ( sessionId SERIAL, mentorId INTEGER , menteeId INTEGER,  mentor_name  VARCHAR(50) NOT NULL, questions  TEXT , menteeName VARCHAR(255) UNIQUE NOT NULL, menteeEmail VARCHAR(50) UNIQUE NOT NULL , status VARCHAR(50) NOT NULL  ,PRIMARY KEY (sessionId));
+   
+     CREATE TABLE IF NOT EXISTS reviews (  sessionId SERIAL, mentorId INTEGER , menteeId INTEGER, remark  TEXT NOT NULL, menteeFullname  TEXT  , score INTEGER,PRIMARY KEY (sessionId));
```
after creating tables, please add at least one instance of dataset for users, mentors, admin, and  sessions

ramadhanbridge committed:
chore(dummy): add dummy data

with this commit, I will be adding default users
 in database

[finishes #168364828]
